packages/fetchai/agents/aries_alice/aea-config.yaml

```diff
@@ -3,27 +3,29 @@ author: fetchai
 version: 0.7.0
 description: An AEA representing Alice in the Aries demo.
 license: Apache-2.0
-aea_version: '>=0.5.0, <0.6.0'
```

Comment:
Please revert; this was correct as it was

Reply:
Done